centos2ol_wrapper.sh

```diff
@@ -0,0 +1,407 @@
+#!/bin/bash
+# centos2ol_wrapper -- Wrapper with sanity checks for centos2ol
+# Copyright Nikolai Bezroukov, 2020. Licensed under Perl Artistic license.

```

Comment:
This should be `Copyright (c) 2021, Oracle and/or its affiliates.` and licensed under the Universal Permissive License (UPL-1.0), as you have signed the Oracle Contributor Agreement which provides shared ownership of copyright.